articles/azure/azs-how-create-vm.md

```diff
@@ -34,62 +34,55 @@ VMs provide the basic compute building blocks in Azure Stack. You can create VMs
 
    For more detailed instructions, see the [*Getting Started Guide for UKCloud for Microsoft Azure*](azs-gs.md).
 
-2. In the favourites panel, select **New**.
+2. In the favourites panel, select **Create a resource**.
 
     ![New option in favourites panel](images/azsp_newmenu.png)
 
 3. In the *New* blade, select **Compute**.
 
     ![Compute option in New blade](images/azsp_newblade.png)
 
-4. In the *Compute* blade, select the image that you want to use for your VM, for example, **Ubuntu Server 17.0**.
+4. In the *Compute* blade, select the template that you want to use for your VM.
 
     ![List of VM images in Compute blade](images/azsp_computeblade.png)
 
-5. In the *Deployment* blade, click **Create** to confirm your VM image selection.
-
-    ![Deployment blade](images/azsp_deploymentblade.png)
-
-6. In the *Create virtual machine* blade, in the *Basics* step, enter general information about the VM, including a name, credentials and resource group, then click **OK**.
+5. In the *Create virtual machine* blade, in the *Basics* step, enter general information about the VM, including a name, credentials and resource group, then click **OK**.
 
     ![Create virtual machine > Basics](images/azsp_createvm_basics.png)
 
-7. In the *Size* step, select the appropriate size for your VM, depending on its purpose, then click **Select**.
+6. In the *Size* step, select the appropriate size for your VM, depending on its purpose, then click **Select**.
 
     For information about the different available VM sizes, see <https://docs.microsoft.com/en-gb/azure/azure-stack/user/azure-stack-vm-sizes>
 
-    > [!TIP]
-    > By default, the list shows a selection of recommended VM sizes. To see all available VM sizes, click **View all**.
-
     ![Create virtual machine > Size](images/azsp_createvm_size.png)
 
-8. In the *Settings* step, change any of the optional settings as required for your VM, then click **OK**.
+7. In the *Settings* step, change any of the optional settings as required for your VM, then click **OK**.
 
     ![Create virtual machine > Settings](images/azsp_createvm_settings.png)
```

Comment:
Can you just check that any sensitive info is blurred. Not sure if we should be showing stuff like subnet IP and the name of the storage account, etc.

Reply:
Have uploaded one where the storage account is removed. Subnet doesn't actually exist as I didn't actually create the network.